changelog/fragments/01-olm-install-retry.yaml

```diff
@@ -0,0 +1,35 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      Fix a bug where `olm install` command is failed fo "no-match" error.
+      
+      The output in this case is something like:
+      
+      ```
+      $ operator-sdk olm install --verbose
+      DEBU[0000] Debug logging is set                         
+      INFO[0000] Fetching CRDs for version "latest"           
+      INFO[0000] Fetching resources for resolved version "latest" 
+      INFO[0001] Creating CRDs and resources                  
+      INFO[0001]   Creating CustomResourceDefinition "catalogsources.operators.coreos.com" 
+      INFO[0001]   Creating CustomResourceDefinition "clusterserviceversions.operators.coreos.com" 
+      INFO[0001]   Creating CustomResourceDefinition "installplans.operators.coreos.com" 
+      INFO[0001]   Creating CustomResourceDefinition "olmconfigs.operators.coreos.com" 
+      INFO[0001]   Creating CustomResourceDefinition "operatorconditions.operators.coreos.com" 
+      INFO[0001]   Creating CustomResourceDefinition "operatorgroups.operators.coreos.com" 
+      INFO[0001]   Creating CustomResourceDefinition "operators.operators.coreos.com" 
+      INFO[0001]   Creating CustomResourceDefinition "subscriptions.operators.coreos.com" 
+      INFO[0001]   Creating Namespace "olm"                   
+      INFO[0001]   Creating Namespace "operators"             
+      INFO[0001]   Creating ServiceAccount "olm/olm-operator-serviceaccount" 
+      INFO[0001]   Creating ClusterRole "system:controller:operator-lifecycle-manager" 
+      INFO[0001]   Creating ClusterRoleBinding "olm-operator-binding-olm" 
+      INFO[0001]   Creating OLMConfig "cluster"               
+      FATA[0001] Failed to install OLM version "latest": failed to create CRDs and resources: no matches for kind "OLMConfig" in version "operators.coreos.com/v1" 
```

Comment:
Nit: I would recommend only showing the full error message to keep the changelog entry as short as possible but still informative